src/backend/core/migrations/00013_fix_content_types.py

```diff
@@ -0,0 +1,100 @@
+import magic
+from django.db import migrations
+from django.core.files.storage import default_storage
+
+def fix_attachments_content_type(apps, schema_editor):
```

Comment:
All of this should be made in a management command. It is a not related to a database modification and tests should be added IMO

Reply:
And so we have to create a job in the helm chart to run this kind of management command. I will made it in this branch if you are ok

Reply:
Yes feel free to do it. 🙏